packages/amplify-graphql-model-transformer/src/__tests__/model-transformer.test.ts

```diff
@@ -1681,30 +1682,4 @@ describe('ModelTransformer:', () => {
       });
     });
   });
-
-  it('should fail if multiple SQL engine types are used', () => {
```

Comment:
Should we change this test to expect to not throw rather than removing it?

Reply:
The new combine-synthesis tests do that, so this doesn't add any particular value.